eng/Versions.props

```diff
@@ -37,7 +36,7 @@
     <MicrosoftDotNetXUnitConsoleRunnerPackageVersion>2.5.1-beta.19406.7</MicrosoftDotNetXUnitConsoleRunnerPackageVersion>
     <MicrosoftDotNetBuildTasksPackagingPackageVersion>1.0.0-beta.19406.7</MicrosoftDotNetBuildTasksPackagingPackageVersion>
     <MicrosoftDotNetRemoteExecutorPackageVersion>1.0.0-beta.19406.7</MicrosoftDotNetRemoteExecutorPackageVersion>
-    <MicrosoftDotNetBuildTasksFeedPackageVersion>2.2.0-beta.19406.7</MicrosoftDotNetBuildTasksFeedPackageVersion>
+    <MicrosoftDotNetBuildTasksFeedVersion>2.2.0-beta.19406.7</MicrosoftDotNetBuildTasksFeedVersion>
```

Comment:
Arcade uses of this property doesn't use the extended form (`PackageVersion`), so in order for `-publish` target to work and find the right package targets we need to call it in the short form. After this PR goes in, I will send a PR to update all this properties and make them use the short form to be consistent.

Reply:
I noticed the same. I made changes to that in my arcade CoreFxTesting PR as well. I recommend to wait until that one is in before changing all the properties.

Reply:
Sounds good. Let me know whenever we can change them

Reply:
It would be nice if we could change this to use the `PackageReference Update="Name" Version="x.x.x.x"` form instead. That is a bit nicer and doesn't require you to come up with various names (cc. @davkean).

https://github.com/dotnet/project-system/blob/0774bafd7ce7ae04039ef80e583d2beb9844bc39/build/Packages.targets

I understand that would require more extensive arcade changes though...

Reply:
@tannergooding if you think that's a good change the correct place to discuss it is to open an issue in Arcade and tag @jcagme.